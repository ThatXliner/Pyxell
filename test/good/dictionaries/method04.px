
d = {c: c.code for c in 'a'..'z'}

print d.pop('B')
print d.pop('b') ?? -1
print d.pop('b') ?? -1

print d.length
d.clear()
print d
