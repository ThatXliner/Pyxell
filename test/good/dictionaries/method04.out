null
98
-1
25
{:}
